### PR TITLE
UX: Render wizard previews for high-DPI displays

### PR DESCRIPTION
**Pull request by davidtaylorhq**
Sets up a canvas element of twice the required dimensions, scales the coordinate system by 2x, then shrinks the display in css.

Before:
Screenshot 2019-04-12 at 15 29 02

After:
Screenshot 2019-04-12 at 15 28 53

### CODE REVIEW

**Comment by coding-horror** [comment]
ohhh nice! Much better! Thank you David! 😍

File: app/assets/javascripts/wizard/lib/preview.js.es6 [hunk 1]
```diff
@@ -18,10 +18,11 @@ function canvasFor(image, w, h) {
   h = Math.ceil(h);
 
   const can = document.createElement("canvas");
-  can.width = w;
-  can.height = h;
+  can.width = w * 2;
```
Comment (eviltrout) [inline app/assets/javascripts/wizard/lib/preview.js.es6]: Shouldn't this be `window.devicePixelRatio` ? It would be 2 on a hi dpi display, 1 otherwise.

Comment (eviltrout) [reply app/assets/javascripts/wizard/lib/preview.js.es6]: (same for all the other hard coded 2s below!)

Comment (davidtaylorhq) [reply app/assets/javascripts/wizard/lib/preview.js.es6]: Oh, I didn't know that API existed 😍

**Comment by SamSaffron** [comment]
awesome change!

**Pull request merged**